src/javascript/EditPanel/WorkInProgress/Chip/WipInfo.Chip.spec.js

```diff
@@ -34,13 +36,15 @@ describe('WipInfoChip', () => {
     });
 
     it('Should display chip when showChipHeader is true', () => {
-        let wrapper = shallow(<WipInfoChip/>);
+        let RenderWrapper = shallow(<WipInfoChip/>).props().render;
+        let wrapper = shallow(<RenderWrapper field={field}/>);
         expect(wrapper.debug()).toContain('Chip');
     });
 
     it('Should display chip when showChipHeader is false', () => {
         showChip = false;
-        let wrapper = shallow(<WipInfoChip/>);
+        let RenderWrapper = shallow(<WipInfoChip/>).props().render;
```

Comment:
```suggestion
        const RenderWrapper = shallow(<WipInfoChip/>).props().render;
```

Reply:
you can change that everywhere if you want :)

Reply:
let's stick to the current changes :)